components/gfx_traits/print_tree.rs

```diff
@@ -50,10 +59,25 @@ impl PrintTree {
         }
     }
 
+    fn print_level_child_prefix(&self) {
+        for _ in 0..self.level + 1 {
+            print!("\u{2502}  ");
+        }
+        print!("       ");
```

Comment:
Consider using `repeat` so you can adjust how many spaces easily in the future?
https://stackoverflow.com/questions/31216646/repeat-string-with-integer-multiplication